mrtt-ui/src/library/mapDataForApi.js

```diff
@@ -0,0 +1,42 @@
+import { questionMapping } from '../data/questionMapping'
+import separateOutliersFromInliers from './separateOutliersFromInliers'
+
+const getFormattedAnswer = (questionMetaData, uiAnswer) => {
+  const { questionType, options } = questionMetaData
+
+  if (questionType === 'singleChoice') {
+    return [{ choice: uiAnswer }]
+  }
```

Comment:
formatted this based on [Angelo's question types doc](https://docs.google.com/presentation/d/1wCnIHgEHdsb-3BEoAlOaCzEXi3woC_6KKT_qJXVj2-k/edit#slide=id.g12683deca5e_0_23). Im feeling lets follow that until it doesnt work for us if we reach that point. He has put a lot of thought into it, so its a good starting place (or ending place)

Reply:
I don't think it works, sorry. I don't want to hack apart the react hook form and mutate the single choice options then pop them back in. It's an usual format and think we need him to be flexible in this case.

Reply:
Is it a blocker for you?